Gemfile

```diff
@@ -22,7 +22,7 @@ gem "govuk_frontend_toolkit", "0.44.0"
 if ENV["GOVSPEAK_DEV"]
   gem "govspeak", path: "../govspeak"
 else
-  gem "govspeak", "3.4.0"
+  gem "govspeak", "~> 3.5"
```

Comment:
Sorry, should have mentioned that in the comment. Yes this is intentional because I saw there were quite a few times in the commit history where we have to bump the version, and I don't think any minor updates can be breaking changes so I thought this will be simpler for the future. But I'm open to comments on this point.